### PR TITLE
Docs: Adding example introductory paragraph

### PR DESCRIPTION
**Pull request by sthetland**
Adding a description to the Terraform code sample for creating resource groups v2. 

I'm also changing the operator of the main group to what _I think_ it's meant to be... please check me on this.

### CODE REVIEW

**Review by zekisherif — APPROVED** [review]

File: website/docs/r/resource_group.html.markdown [hunk 1]
```diff
@@ -74,17 +76,18 @@ The following arguments are supported:
...
 
-A Lacework Resource Group can be imported using a `RESOURCE_GROUP_GUID`, e.g.
+You can import a Lacework resource group by `RESOURCE_GROUP_GUID`, for example:
 
 ```
 $ terraform import lacework_resource_group.example EXAMPLE_1234BAE1E42182964D23973F44CFEA3C4AB63B99E9A1EC5
 ```
--> **Note:** To retrieve the `RESOURCE_GROUP_GUID` from existing resource groups in your account, 
+
+**Note:** To retrieve the `RESOURCE_GROUP_GUID` from existing resource groups in your account,
```
Comment (afiune) [inline website/docs/r/resource_group.html.markdown:91]: Let's revert this change since the Sigil `->` will render a box on the docs. Like this one https://registry.terraform.io/providers/lacework/lacework/latest/docs/resources/integration_gcr#credentials

Screenshot 2023-08-30 at 10 05 46 AM

More information about Terraform provider documentation can be found here:
https://developer.hashicorp.com/terraform/registry/providers/docs#callouts
